src/components/Heading/Heading.tsx

```diff
@@ -52,6 +58,9 @@ export const Heading = chakra(
       id,
       level = "two",
       size,
+      isCapitalized,
```

Comment:
Please alphabetize the list of props.  It looks like one or two may have been out of order to begin with.